airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/OssMetricsRegistry.java

```diff
@@ -103,6 +103,12 @@ public enum OssMetricsRegistry implements MetricsRegistry {
       MetricEmittingApps.METRICS_REPORTER,
       "num_total_scheduled_syncs_last_day",
       "number of total syncs runs in last day."),
+
+  NUM_UNUSUALLY_LONG_SYNCS(
+      MetricEmittingApps.METRICS_REPORTER,
+      "num_unusually_long_syncs",
+      "number of unusual long syncs compared to their historic performance."),
+
   OLDEST_PENDING_JOB_AGE_SECS(MetricEmittingApps.METRICS_REPORTER,
```

Comment:
there are a few metrics further down that aren't sorted. I know this isn't in your changeset - do you mind sorting them? Thanks!

Reply:
done